### PR TITLE
added pip config

### PR DESCRIPTION
**Pull request by wouterdb**
Added pip config

### CODE REVIEW

File: {{cookiecutter.project_name}}/project.yml [hunk 1]
```diff
@@ -7,9 +7,21 @@ copyright: {{cookiecutter.copyright}}
 modulepath: libs
 downloadpath: libs
 repo:
-    - type: package
-      url: {{cookiecutter.package_repo_url}}
     - type: git
       url: {{cookiecutter.repo_url}}
+    # Index url for V2 modules
+    # This is for <=ISO6
```
Comment (sanderr) [inline {{cookiecutter.project_name}}/project.yml]:
```suggestion
    # This is for <=ISO6 (inmanta-core<=10)
```

File: {{cookiecutter.project_name}}/project.yml [hunk 2]
```diff
@@ -7,9 +7,21 @@ copyright: {{cookiecutter.copyright}}
...
+    - type: package
+      url: {{cookiecutter.package_repo_url}} 
+pip: # for >=ISO7
```
Comment (sanderr) [inline {{cookiecutter.project_name}}/project.yml]:
```suggestion
pip: # for >=ISO7 (inmanta-core>=11)
```

File: {{cookiecutter.project_name}}/project.yml [hunk 3]
```diff
@@ -7,9 +7,21 @@ copyright: {{cookiecutter.copyright}}
...
+    # This is for <=ISO6
+    - type: package
+      url: {{cookiecutter.package_repo_url}} 
+pip: # for >=ISO7 
+  # Index url for python package and V2 modules
+  index_url: {{cookiecutter.package_repo_url}}
+  # Extra index urls for python package and V2 modules
+  extra_index_url: []
+  # Allow pip to install prerelease versions or not?
+  pre: {{cookiecutter.install_mode != "release"}}
```
Comment (sanderr) [inline {{cookiecutter.project_name}}/project.yml:22]: I don't love it but here I think we may not have a choice if we want to keep it simple.

**Review by sanderr — CHANGES_REQUESTED** [review]
Two tiny change suggestions.

File: {{cookiecutter.project_name}}/project.yml (hunk 3 above)
Comment (wouterdb) [reply {{cookiecutter.project_name}}/project.yml:22]: I agree

**Review by sanderr — APPROVED** [review]

**Review by FloLey — APPROVED** [review]

**Pull request merged**